src/main/java/com/redislabs/redisai/Script.java

```diff
@@ -104,6 +107,7 @@ public String getSource() {
     return source;
   }
 
+  @Deprecated
```

Comment:
why?

Reply:
@DvirDukhan Source is a mandatory param for a Script. I'm thinking about setting it from only constructor.